lib/JumpScale/baselib/atyourservice/AtYourServiceFactory.py

```
@@ -66,11 +66,12 @@ def reset(self):
         self._repos = {}
         j.dirs._ays = None
 
+
     @property
     def repos(self):
         if self._repos == {}:
             for path in j.atyourservice.findAYSRepos():
-                name = j.sal.fs.getBaseName(path)
+                name = path.strip('/').replace('/', '.')
```

Let's raise an error when trying to get a repo with name when multiple repos with the same name exist.
Error should be helpful. Something like "multiple AYS repos with name "%s" are found under locations " " and " ". Please use j.atyourservice.get(path=<path>) instead"